### ISSUE TITLE
shipmonk.publicPropertyNotReadonly does not support asymmetric visibility

### ISSUE THREAD

**Issue by JanTvrdik** [shipmonk-rnd/phpstan-rules#319]
No error should be reported, when `public private(set)` is used, e.g.

```php
#[Entity]
#[Table(name: '`user`')]
#[UniqueConstraint(columns: ['email'])]
class User
{

    #[Id]
    #[Column(type: UuidType::NAME)]
    public private(set) Uuid $id;

    #[Column(type: Types::TEXT)]
    public private(set) string $email;

}
```
